### ISSUE TITLE
11 sentences which are just "Understand"

### ISSUE THREAD

**Issue by AngledLuffa** [UniversalDependencies/UD_English-GUM#79]
The training dataset has 11 sentences which are each just one word, `Understand`.  If there's any room to filter things like that, it would be useful to limit it to make just one (or even zero, since normally I would expect `Understood` or `Understand?`)

**Comment by amir-zeldes** [comment]
Exactly, one of the genres is travel guides from Wikivoyage, and it's part of their basic template. They don't all have all sections, but some of the suggested sections are "See", "Do", "Understand" and "Get in", so many of those have just one word. It does represent the original text, so it should stay that way (and quantitatively these sentences are pretty negligible)
